golang/Makefile

```diff
@@ -1,16 +1,17 @@
-PACKAGE := github.com/dyrector-io/dyrectorio/agent
-CONTAINER_URL := ghcr.io/dyrector-io/dyrectorio/agent/
+PACKAGE := github.com/dyrector-io/dyrectorio/golang
+ORG_REGISTRY := ghcr.io/dyrector-io/dyrectorio/
```

Comment:
IMHO Put the slash end of the ENV is a bad small, the next line looks terrible.

Reply:
These are not env varriables. I prefer that one too, even wondered why we did not opt for that initially. Replacing.